tock/utilization/tests/test_views.py

```diff
@@ -0,0 +1,127 @@
+import datetime
+
+from django.test import TestCase
+from django.core.urlresolvers import reverse
+from django.contrib.auth.models import User
+from django.test.client import Client
+
+from django_webtest import WebTest
+
+
+from ..views import get_fy_first_day, get_dates
+from ..utils import get_fy_first_day, get_dates, calculate_utilization
+from hours.models import ReportingPeriod, Timecard, TimecardObject
+from projects.models import Project, AccountingCode, Agency
+from employees.models import UserData
+
+
+FIXTURES = [
+    'tock/fixtures/prod_user.json',
+    'projects/fixtures/projects.json',
+    'hours/fixtures/timecards.json',
+    'employees/fixtures/user_data.json',
+]
+
+
+class TestUtils(TestCase):
+    fixtures = FIXTURES
+
+    def test_fy_first_date(self):
+        date = datetime.date(2014, 7, 4)
+        self.assertEqual(get_fy_first_day(date), datetime.date(2013, 10, 1))
+
+    def test_get_dates(self):
+        periods = len(ReportingPeriod.objects.all())-1
```

Comment:
I caught this in a few spots in the code; if you find yourself looking to get the number of items in a QuerySet, use the QuerySet `count()` method instead, e.g. in this case:

`ReportingPeriod.objects.count() - 1`

When you use `len(ReportingPeriod.objects.all())`, you'll evaluate the whole QuerySet and since this is returning all models, that could be quite an expensive query and large result set!  Let the database do the heavy lifting here. :-)

For more info:  https://docs.djangoproject.com/en/1.10/ref/models/querysets/#count